examples/NonArduino/Tock/tock.cmake

```diff
@@ -0,0 +1,30 @@
+## License:  Revised BSD License, see LICENSE.TXT file included in the project
```

Comment:
RadioLib license is MIT, this is most likely referencing a different file. If so, please include this license in the example.